contract/src/platform-goals/start-contract.js

```diff
@@ -35,6 +35,7 @@ export const installContract = async (
  *   startArgs?: StartArgs;
  *   issuerNames?: string[];
  * }} opts
+ * @param {object} [privateArgs]
```

Comment:
I tried to get rid of this copy of `privateArgs`. There's already a `privateArgs` in `startArgs`.

Reply:
oic. I'll revise